src/utils/constants.ts

```diff
@@ -0,0 +1,10 @@
+export const SCREEN_WIDTH = {
```

Comment:
I have created two new files to help maintain the codebase as we start to add new features.
Hope you don't mind.

Reply:
YAGNI it is.
Addressed in 3795d3f0f7fc3ea55fa895337eda048363f3578b